<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QCalendarWidget" name="calendarWidget"/>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="keepTimeCheckBox">
     <property name="text">
      <string>keep time</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="useCheckBox">
     <property name="text">
      <string>change timestamp</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDateTimeEdit" name="dateTimeEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="keepDateCheckBox">
     <property name="text">
      <string>keep date</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>calendarWidget</sender>
   <signal>activated(QDate)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTimeEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>calendarWidget</receiver>
   <slot>setSelectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
